internal/subshell/sscommon/rcfile.go

```diff
@@ -348,7 +348,11 @@ func SetupProjectRcFile(prj *project.Project, templateName, ext string, env map[
 	pathList, ok := env["PATH"]
 	inPathList, err := fileutils.PathInList(listSep, pathList, currExecAbsDir)
 	if !ok || !inPathList {
-		rcData["ExecAlias"] = currExec // alias {ExecName}={ExecAlias}
+		safeExec := currExec
+		if strings.Index(currExec, " ") != -1 {
+			safeExec = fmt.Sprintf(`"%s"`, currExec) // quote for alias
+		}
+		rcData["ExecAlias"] = safeExec // alias {ExecName}={ExecAlias}
```

Comment:
I believe that `strings.Index` returns `-1` if the substrings _isn't_ found. So this logic seems backwards.

You could replace the function call here with `strings.ContainsAny(curreExec, " ")` or with a simple regex like `\s`

Reply:
I did a double-take as well. In this case, if there is a space found, we want to quote the executable (e.g. `"/path to state.exe"`). Otherwise, it can remain unquoted.